tests/qa_automation/qaset_post_patch_run.pm

```diff
@@ -0,0 +1,140 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+package qaset_post_patch_run;
+
+# Summary: Simplify test reporting for qa_automation
+#
+# This execution is entirely based on qa_automation/qa_run. The test result is simplfied to only show failed testcases
+# triggered by regression bug. That will satisfy the need for maintenance update test
+# Using YAML_SCHEDULE to schedule is recommanded to keep flexibility.
+# qaset_pre_patch_run, patch_and_reboot and qaset_post_patch_run should schedule in order.
+#
+# features:
+# 1. Only failed test case names triggered by regression are show on result page.
+# 2. QADB url of comparison of before update and after is posted under testcase name.
+# 3. Added var SOFTFAIL_TESTCASES which make a failed testcase softfailed to avoid triggering
+#    entire test failed until the corresponding bug is fixed.
+# 4. Auto define test in qaset if it's no defined.
+# 5. disble/enable QADB submission with var DISABLE_SUBMIT_QADB.
+#
+# Maintainer: Tony Yuan <tyuan@suse.com>
+
+use strict;
+use warnings;
+use base "qa_run";
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils;
+
+# Create qaset/config file, reset qaset, and start testrun
+sub start_testrun {
+    my $self = shift;
```

Comment:
you are not using `$self` so I suggest to just define it as self-standing function, i.e. delete this line and not call as `$self->start_testrun()` but just `start_testrun()`

Reply:
fixed.